src/lib/src/select-item.directive.ts

```diff
@@ -31,6 +31,7 @@ export class SelectItemDirective implements OnInit, DoCheck {
 
   ngOnInit() {
     if (isPlatformBrowser(this.platformId)) {
+      this.calculateBoundingClientRect();
       setTimeout(() => this.calculateBoundingClientRect());
```

Comment:
Can be removed if you refactor the code to what I have suggested above. But I am of course open for discussion and my feedback shouldn't be taken for granted. If you have another idea, please don't hesitate to post it here so that we can openly discuss all sorts of ideas.